package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@react-ui-org/react-ui",
-      "version": "0.47.0",
```

Comment:
Why is this change here? It should have been already changed during the release, shouldn't it? (cc @adamkudrna)

If it fixes something, please do it in separate commit.

Reply:
It came from a regular npm install, I am not sure why it is there either.